DevExtreme CLI: add React Next.js info (#7295)
DevExpress/devextreme-documentation: concepts/50 React Components/50 Application Template/00 Application Template.md

> The DevExtreme React Application Template quickly sets up a new React project. The template includes:

Reviewer: I'm not sure I agree since it does not set up a React project by itself - you use it to set it up

```suggestion
The DevExtreme React Application Template facilitates setting up a new React project. The template includes:
```

or

```suggestion
The DevExtreme React Application Template expedites setting up a new React project. The template includes:
```

Author: What if we rephrase this bit and say "simplifies the setup process" to avoid gerund verbs? Also now that I look at this sentence I think using "React projects" instead of "a React project" will simplify it just a touch further:
```suggestion
The DevExtreme React Application Template simplifies the setup process of new React projects. The template includes:
```

Reviewer: Okay, but now you are adding more words just to avoid a gerund. I can't imagine you'd want to say "simplifies the setup process of something" if you were describing this to someone in conversation, you'd instead say

```suggestion
The DevExtreme React Application Template simplifies setting up a new React project. The template includes:
```

which is similar to the other versions I suggested.

- Multiple sample views 
- DevExtreme UI Widgets: [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Overview/), [Form](/Documentation/Guide/UI_Components/Form/Overview/), [TreeView](/Documentation/Guide/UI_Components/TreeView/Getting_Started_with_TreeView/), and others

> - Placeholder authentication functions (so you can implement a secure log-in system)

Reviewer: ```suggestion
- Placeholder authentication functions (to implement a secure login system)
```

> The DevExtreme React Application Template allows you to speed up the process of setting up a new React project that integrates the DevExtreme UI library. The template includes multiple components such as [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Overview/), [Form](/Documentation/Guide/UI_Components/Form/Overview/), [TreeView](/Documentation/Guide/UI_Components/TreeView/Getting_Started_with_TreeView/), and more. Multiple sample views are included, as well as placeholder authentication functions for implementing a secure log-in system.

Reviewer 2: The first sentence is rather complicated. You can just look at the number of actions: "allows", "speed up", "setting up", "integrates". And all of this describes a pretty simple fact that's clear from the name. "DevExtreme React Application Template" quickly creates a React Application with DevExtreme stuff in it. 

Then you list everything that's included in the next couple of sentences. I would just list it in a list :-) 

```suggestion
The DevExtreme React Application Template quickly sets up a new React project. The template includes:

- Multiple sample views 
- DevExtreme UI Widgets: [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Overview/), [Form](/Documentation/Guide/UI_Components/Form/Overview/), [TreeView](/Documentation/Guide/UI_Components/TreeView/Getting_Started_with_TreeView/), and others
- Placeholder authentication functions (so you can implement a secure log-in system)
```
